integration-tests/go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/slack-go/slack v0.12.2
 	github.com/smartcontractkit/chainlink-automation v1.0.3
 	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240516150131-e1be553a9d10
-	github.com/smartcontractkit/chainlink-testing-framework v1.28.12
+	github.com/smartcontractkit/chainlink-testing-framework v1.28.14-0.20240517133715-c48852fb81f5
```

Comment:
Will need to cut a new version for this probably

Reply:
yup, just wanna merge one more thing to the CTF before releasing a new verison